solang-parser/src/lexer.rs

```diff
@@ -605,7 +604,7 @@ impl<'input> Lexer<'input> {
             start -= 1;
```

Comment:
You can remove line 601 and make `start` a mutable parameter.

Reply:
We use start again on line 607, so this would not work

Reply:
I don't understand why not. You create a new variable with the same name on line 601.

Reply:
You're right, fixed